spec/support/activity_form.rb

```diff
@@ -9,8 +9,12 @@ def initialize(activity:, fund:, level:)
     @level = level
   end
 
-  def complete!
-    send("fill_in_#{fund}_#{level}_activity_form")
+  def complete!(is_oda: nil)
+    if is_oda.nil?
+      return send("fill_in_#{fund}_#{level}_activity_form")
+    end
+
+    send("fill_in_#{fund}_#{level}_activity_form", is_oda: is_oda)
```

Comment:
I wonder if this should be the other way around (`if is_oda ... end ... default`), given `nil` is the default value?

Reply:
Have left this as-is for now.